code/modules/hydroponics/hydroponics.dm

```diff
@@ -132,7 +132,7 @@
 	else if(istype(Proj , /obj/projectile/energy/florayield))
 		return myseed.bullet_act(Proj)
 	else if(istype(Proj , /obj/projectile/energy/florarevolution))
-		if(myseed)
+		if(myseed?.mutatelist)
```

Comment:
Substitute `if(myseed?.mutatelist)` and `if(myseed.mutatelist.len > 0)` for `if(length(myseed?.mutatelist))`